### ISSUE TITLE
Unable to run on a fresh install, "Unable to find an inventory file"

### ISSUE THREAD

**Issue by OldhamMade** [spencergibb/battleschool#37]
Trying to use battleschool on a brand new install, but I'm getting the error below. Steps I followed to get to this point:

* install Xcode command-line tools
* `curl -Ls https://bootstrap.pypa.io/get-pip.py | sudo python`
* `sudo pip install battleschool`

Then (using the example file from the docs):

```
[mac:~ home]$ battle --config-file https://db.tt/aG2uyydU
Traceback (most recent call last):
  File "/usr/local/bin/battle", line 7, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/Library/Python/2.7/site-packages/battleschool/main.py", line 118, in main
    inventory = ansible.inventory.Inventory(options.inventory)
  File "/Library/Python/2.7/site-packages/ansible/inventory/__init__.py", line 139, in __init__
    raise errors.AnsibleError("Unable to find an inventory file, specify one with -i ?")
ansible.errors.AnsibleError: Unable to find an inventory file, specify one with -i ?
```

Unable to progress past this point. 

`/usr/share/battleschool/defaults/hosts` doesn't exist, actual location is `/Library/Python/2.7/site-packages/usr/share/battleschool/defaults/hosts`

**Comment by spencergibb** [comment]
@OldhamMade when I install fresh, the file gets put into `/usr/share/battleschool/defaults/hosts`.  What does the output of `env` look like?

**Comment by OldhamMade** [comment]
`env` says the user is me, and I've also tried installing from a version of Python & pip installed via homebrew, but I'm still getting the same issues.

I've worked round the problem by symlinking `/Library/Python/2.7/site-packages/usr/share/battleschool/defaults/hosts` to `/usr/share/battleschool/defaults/hosts` after installing battleschool.

**Comment by fearoffish** [comment]
Fresh El Capitan install. Installed Homebrew, installed python, pip installed battleschool.   I get the same as above.

**Comment by echohack** [comment]
+1, I think this might be related to the SIP https://en.wikipedia.org/wiki/System_Integrity_Protection

**Comment by vladdancer** [comment]
@fearoffish I think it's not a big issue. You can fix this on fresh install on El Capitan with installed battle school:
```bash
cd ~ && mkdir .battleschool
cat /usr/local/lib/python2.7/site-packages/usr/share/battleschool/defaults/hosts > .battleschool/hosts
battle -i .battleschool/hosts
```

**Comment by spencergibb** [comment]
fixed via ae22fb613d5465973e1a0437bfc484f24cb90e01
